google/resource_compute_instance_test.go

```diff
@@ -282,6 +282,40 @@ func TestAccComputeInstance_diskEncryption(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstance_kmsDiskEncryption(t *testing.T) {
+	t.Parallel()
+
+	var instance compute.Instance
+	var instanceName = fmt.Sprintf("instance-test-%s", acctest.RandString(10))
+	bootKmsKeyName := "projects/project/locations/us-central1/keyRings/testing-cloud-kms/cryptoKeys/key-0/cryptoKeyVersions/1"
+	diskNameToEncryptionKey := map[string]*compute.CustomerEncryptionKey{
```

Comment:
It looks like the actual config that this calls only looks at the RawKey in this block. I'd recommend creating a completely new test to test KMS keys, especially since we also tend to create new projects for those tests (see https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_kms_crypto_key_test.go)